composer.json

```diff
@@ -4,12 +4,12 @@
     "type": "drupal-module",
     "license": "GPL-2.0-or-later",
     "require": {
-        "dpc-sdp/tide_core": "^1.2.5",
-        "dpc-sdp/tide_media": "^1.2.5"
+        "dpc-sdp/tide_core": "^1.2.6",
+        "dpc-sdp/tide_media": "^1.2.6"
     },
     "suggest": {
-        "dpc-sdp/tide_api:^1.2.6": "Allows to use Drupal in headless mode",
-        "dpc-sdp/tide_webform:^1.1.6": "Allows to use questionnaires"
+        "dpc-sdp/tide_api:^1.2.11": "Fixes cache issue.",

```

Comment:
@vincent-gao what is the reason for changing the description of the suggested modules?

Reply:
solved.